services/httpd/handler.go

```diff
@@ -555,13 +555,17 @@ func serveExpvar(w http.ResponseWriter, r *http.Request) {
 
 // HttpError writes an error to the client in a standard format.
 func HttpError(w http.ResponseWriter, err string, pretty bool, code int) {
+	w.Header().Set("Content-Type", "application/json")
```

Comment:
This is unnecessary, there is something that is already taking care of this, and it collides with that.